frontend/src/assets/i18n/de.json

```
@@ -123,10 +123,15 @@
     }
   },
   "DIALOG_ERRORS": {
+    "REQUIRED": "{{fieldName}} ist ein Pflichtfeld.",
+    "MINLENGTH": "{{fieldName}} muss mindestens {{requiredLength}} Zeichen lang sein.",
+    "MAXLENGTH": "{{fieldName}} darf maximal {{requiredLength}} Zeichen lang sein.",
+    "PATTERN": "{{fieldName}} muss eine Zahl sein.",
```

```suggestion
    "NUMBER": "{{fieldName}} muss eine Zahl sein.",
```
If this is only for the numbers, then the name should communicate that clearly.